app/dashboard/views.py

```diff
@@ -820,33 +822,42 @@ def users_fetch(request):
     all_users = []
     for user in all_pages.page(page):
         profile_json = {}
-        profile_json = user.to_standard_dict()
-        if user.avatar_baseavatar_related.exists():
-            user_avatar = user.avatar_baseavatar_related.first()
-            profile_json['avatar_id'] = user_avatar.pk
-            profile_json['avatar_url'] = user_avatar.avatar_url
+        # profile_json = user.to_standard_dict()
         count_work_completed = Activity.objects.filter(profile=user, activity_type='work_done').count()
         count_work_in_progress = Activity.objects.filter(profile=user, activity_type='start_work').count()
-        previously_worked_with = 0
-        if profile:
-            previously_worked_with = BountyFulfillment.objects.filter(
-                bounty__bounty_owner_github_username__iexact=profile.handle,
-                fulfiller_github_username__iexact=user.handle,
-                bounty__network=network,
-                bounty__accepted=True
-            ).count()
-
+        profile_json['id'] = user.id
```

Comment:
`profile_json = {k: user['k'] for k in ['id', 'actions_count', 'created_on'... etc]}` and then `profile_json['job_status'] = user.job_status_verbose if user.job_search_status else None` as the only unique one. Less lines is (almost) always good :)